application/forms/AddNodeForm.php

```diff
@@ -391,6 +438,34 @@ protected function enumServiceList($host)
         return $services;
     }
 
+    protected function enumServiceStateList()
+    {
+        $serviceStateList = array(
+            0 => $this->translate('OK'),
+            1 => $this->translate('WARNING'),
+            2 => $this->translate('CRITICAL'),
+            3 => $this->translate('UNKNOWN'),
+            99 => $this->translate('PENDING'),
+        );
+
+        return $serviceStateList;
+    }
+
+    protected function statesToString()
+    {
+        $stateString = null;
+        foreach ($this->enumServiceStateList() as $state => $stateName) {
+            if ($this->getValue($stateName) != null) {
+                if ($stateString != null) {
+                    $stateString .= ',';
+                }
+                $stateString .= $state. '-'. $this->getValue($stateName);
+            }
+        }
```

Comment:
`==` vs. `===` applies here as well: `!==`

That's especially important if checking for `null`.

Also, please put a space before and after any operator. In this case the `.`s on line no. 462. (Though not the `.=`, that's fine)

Reply:
ok